README.md

```diff
@@ -27,7 +27,13 @@ logo by [@nwlandas](https://twitter.com/nwlandas)
 
 ## Installation
 
-_it is in development and not yet finished_
+### Windows
```

Comment:
Please move the install instructions to the scripts/setup directory
they are for contrubitors